docs/modules/ROOT/pages/operations.adoc

```diff
@@ -174,62 +175,199 @@ The URL _\https://<dbid>.databases.neo4j.io/browser/_ can also be used to access
 
 
 [[results]]
-== Visualizing results
-
-// Graph, Table and Text views.
-// https://neo4j.com/developer/graph-visualization/#_neo4j_browser
+== Visualize results
 
 There are a variety of ways to view data in Neo4j Browser.
-Any queries you run in Neo4j Browser populate your results below the command line.
-Query results are rendered either as a visual graph, a table format, an ascii-table result, and in JSON.
+Any queries you run in Neo4j Browser populate the result frame below the Cypher editor.
```

Comment:
Queries that you run in the Neo4j Browser editor, will populate a reusable result frame.